editor/reducer.js

```diff
@@ -441,13 +441,21 @@ export function mode( state = 'visual', action ) {
 	return state;
 }
 
-export function preferences( state = { isSidebarOpened: ! isMobile }, action ) {
+export function preferences( state = { isSidebarOpened: ! isMobile, panels: { 'post-status': true } }, action ) {
 	switch ( action.type ) {
 		case 'TOGGLE_SIDEBAR':
 			return {
-				... state,
+				...state,
 				isSidebarOpened: ! state.isSidebarOpened,
 			};
+		case 'TOGGLE_SIDEBAR_PANEL':
+			return {
+				...state,
+				panels: {
+					...state.panels,
+					[ action.panel ]: ! get( state, [ 'panels', action.panel ], false ),
```

Comment:
Is there any need for `get` here? `state.panels` seems it should always be an object.

Reply:
It's needed for users who already have the preferences persisted.